docs/docs/how-to/file-uploads.md

```diff
@@ -446,6 +446,50 @@ Now let's add the ability to bring back the uploader if you decide you want to c
 
 We're borrowing the styles from the submit button and making sure that the image has both a top and bottom margin so it doesn't crash into the new button.
 
+## Safe usage of your free plan
+Having a free plan is great, but if you just load images and never actually remove the unnecessary ones, you'll be in trouble.
+
+To avoid this, we'd better implement the `deleteImage` mutation. It will enable you to make a call to the Filestack API to remove your resources, and on success, you will remove the row in the `Image` model.
+
+You are going to need a new `.env` called `REDWOOD_ENV_FILESTACK_SECRET`, which you can find in Filestack > Security > Policy & Signature: App Secret.
+
+Also, let's add this package to the right side of our current concern:
```

Comment:
Hmm I was confused by this as I don't think we've ever used to the term "concerns" before to refer to the sides. I'd just rephrase this to something like "We're going to perform the delete on the server side, so let's add this package to the api side:"